### DISCUSSION TITLE
Creating a Mail app

### DISCUSSION THREAD

**Discussion by leneleaf** [FossifyOrg/General-Discussion#68] [Ideas]
I find that there is a problem with the currently available foss mailing apps on android. All of them tend to be really great, but they are usually a bit more cluttered than I would like them to be. Personally, when I look at emails on my phone, its because I need to do something really simple like verify an account or check to see if someone responded yet. I'm sure I have an unusual usecase, but it would still be nice to have this in the fossify ecosystem.

**Comment by Aga-C** [comment]
Fossify is a fork of Simple Mobile Tools which were always done in a spirit of having no internet access. 

@naveensingh Is it still valid for Fossify?

**Reply by naveensingh** [reply]
Internet permission is probably the most misused permission in all of history so unless the app's core functionality requires Internet permission, we won't add it. That means all of the existing apps will never get access to the internet.

A mail app is not a bad idea for the suite but I think it's too soon to even talk about it.

**Reply by parrenin** [reply]
With all respect to the developers, I am not sure I completely agree with the Fossify policy regarding internet permission.
I understand the value of having no internet permission (for privacy).
On the other hand, if there are useful features which require internet access, it is a shame not to support them.
I would be OK with apps which do not have internet permission by default, but which could be optionally activated to support some internet features (such as RCS chat in Messages or Google Maps address book in Phone). 
If we take the example of the RCS protocol, it is actually better for privacy since it is End-to-End encrypted, so if Fossify values privacy, it should actually support it. 
RCS is a free protocol and the alternatives are proprietary (WhatsApp, Messenger, etc.).

**Reply by chesio** [reply]
@parrenin Android apps cannot optionally activate / request internet permission. If an app needs internet permission, it gets it at install time (no questions asked): https://developer.android.com/develop/connectivity/network-ops/connecting

**Reply by parrenin** [reply]
@chesio OK, thanks for the information, I did not know that.
I would trust more a Fossify app with internet access than a proprietary app.

**Reply by naveensingh** [reply]
> Android apps cannot optionally activate / request internet permission

It's absurd that the internet permission is considered a normal install-time permission, it's a dangerous permission.

@parrenin for RCS, you may subscribe to this discussion: https://github.com/FossifyOrg/Messages/discussions/34.

**Reply by parrenin** [reply]
Than you @naveensingh for your answer, I have subscribed to the RCS issue.

**Reply by parrenin** [reply]
Thinking more about this issue of internet permission, would not it be possible to have two versions of the same app, a 'safe' one and a 'connected' one?
I am not developer, but I imagine it is possible to have compile-time switches to activate/deactivate functionalities.

**Reply by naveensingh** [reply]
It was kind of discussed before: https://github.com/FossifyOrg/Gallery/issues/83#issuecomment-1905575879

Maybe some day :)

**Comment by Mr-Bajs** [comment]
K-9 Mail/Thunderbird are an android client that's open source and have great support. Now that there are decent maintained foss alternatives for email clients, then one could argue that it would be best to focus on apps that doens't have any great foss alternatives in the android app ecosystem.

**Reply by inson1** [reply]
Or just focus on apps we have now, its too much even for one full time person and we dont have funding even for that (or at least I think)

**Reply by naveensingh** [reply]
@inson1 yes, we don't have any funding for sustained development. There are few recurring and one time donations but looking at the donations SMT received publicly, it seems like donations alone won't be enough to go full-time unless the project becomes insanely popular. It's not really an immediate problem, we'll see.

**Comment by inson1** [comment]
I'm for closing this. As now is the question if this fork even survives and not what new workload to add.

**Comment by naveensingh** [comment]
Closed it for now, this is not something we'll do anytime soon.
